index.js

```diff
@@ -11,16 +11,56 @@ if (process.env.NODE_ENV !== 'production') {
     Symbol.for('react.element')) ||
     0xeac7;
 
+  var REACT_PROVIDER_TYPE =
```

Comment:
these should be pulled from react-is and not hardcoded here.

Reply:
I would have to add the `react-is` as a production dependency, are you open to that? I saw you were already hardcoding the element type. I wouldn't mind switching all of them.

Reply:
i'm not a collaborator on this repo, but personally yes, i am - it's minified heavily in production, and you could also wrap the require in a NODE_ENV check.